### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by chris84** [Sixze/ALS-Refactored#264] [Q&A] [answered]
Does anyone perhaps have any more documentation / tutorials on using ALS refactored?

**Comment by murataka** [comment, marked as answer]
It is  best to play with it yourself , add more specific questions after that. A general documentation would be too time consuming to prepare , however , I hope that people ask more questions in these discussions , and maybe after some time , we can use those as a documentation or to prepare documentation. Maybe we can create a discussion post about youtube links / example projects , additional to the the discussions here.  I'm just a contributor/user of als-refactored, so those are just my own ideas.

**Reply by chris84** [reply]
Ok thanks, I will learn as I go :)
